Example.App.js

```diff
@@ -13,20 +13,15 @@ export default class App extends React.Component {
   onMessage = event => {
     if (event && event.nativeEvent.data) {
       if (['cancel'].includes(event.nativeEvent.data)) {
-        // do not call this.captchaForm.hide(); function in this condition. Otherwise app will crash.
+        this.captchaForm.hide();
```

Comment:
Why is this safe now?

Reply:
Due to the previous implementation (which was wrong in my opinion), this call may lead to implicit recursion (infinite calls): `hide -> onMessage -> hide -> onMessage ...`

This PR changes it:
 - all `this.captchaForm.hide();` (hide without arguments) considered as calls from "user code" and will not lead to `onMessage` callback
 - calls `hide` with `source` argument like this `this.captchaForm.hide("back_button");` will lead to `onMessage({nativeEvent:{data: 'cancel'}})`